baremaps-iploc/src/main/java/com/baremaps/iploc/data/IpLocStats.java

```diff
@@ -14,12 +14,62 @@
 
 package com.baremaps.iploc.data;
 
-// TODO: this should be a package private class with non-static fields
+/** Store stats related to the geneation of the IpLoc database from the RIR's files */
 public class IpLocStats {
-  public static int inetnumInsertedByAddress = 0;
-  public static int inetnumInsertedByDescr = 0;
-  public static int inetnumInsertedByCountry = 0;
-  public static int inetnumInsertedByGeoloc = 0;
-  public static int inetnumNotInserted = 0;
-  public static int inetnumInsertedByCountryCode = 0;
+  int insertedByAddressCount = 0;
```

Comment:
These fields should probably be private if you increment their values in methods.